libs/blocks/ost/ost.css

```diff
@@ -0,0 +1,35 @@
+body>header,
```

Comment:
Instead of hiding the header and footer with css, you can actually disable the `header` and `footer` of any milo page using `metadata`, see https://milo.adobe.com/docs/authoring/metadata#. Simply add a `metadata` table in the `milo/tools/ost` document and set both `header` and `footer` to `off`.

Reply:
Thanks @honstar good catch. done.
Updated the after URL.